Demo/Sources/Recycling/GridViews/CompactMarkets/ToriCompactMarketsDemoView.swift

```diff
@@ -5,7 +5,7 @@
 import FinniversKit
 import DemoKit
 
-class CompactMarketsDemoView: UIView, Demoable {
```

Comment:
could you please explain why `CompactMarketsDemoView` was replaced with `ToriCompactMarketsDemoView`
it seems to me that the changes should be additive, and not remove anything